php-laravel-phpcs.md

```diff
@@ -0,0 +1,139 @@
+# Introduction
+
+[PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) is a set of two PHP scripts; the main `phpcs` script that tokenizes PHP, JavaScript and CSS files to detect violations of a defined coding standard, and a second `phpcbf` script to automatically correct coding standard violations. PHP_CodeSniffer is an essential development tool that ensures your code remains clean and consistent.
+
+# Prerequisites
+
+- [PHP 8.1](https://www.php.net/releases/8.1/en.php)
+- [Composer](https://getcomposer.org/)
+- [Laravel 8](https://laravel.com/docs/8.x/releases)
+
+Laravel 8 is used for the development of web applications in PHP.
+
+> Lumen a micro-framework can also be used depending on project need. This document is only verified with Laravel 8.
+
+# Installation
+
+Install the following packages:
+
+- [squizlabs/PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer)
+- [slevomat/coding-standard](https://github.com/slevomat/coding-standard)
+
+**`squizlabs/PHP_CodeSniffe`**
+
+```sh
+composer require squizlabs/php_codesniffer --dev
+```
+
+**`slevomat/coding-standard`**
+
+```sh
+composer require slevomat/coding-standard --dev
+```
+
+You will come across the following query when installing `slevomat/coding-standard`,
+
+```sh
+Do you trust "dealerdirect/phpcodesniffer-composer-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?]
+```
+
+Press `y`
+
+# Configuration
+
+Create `.phpcs.xml` file in the root directory of your project.
+
+Then add the below configuration in this file.
+
+```xml
+<?xml version="1.0"?>
+<ruleset name="PHP_CodeSniffer">
+    <file>app</file>
```

Comment:
there are other folders in Laravel. We may want to enable checks for those too?

![image](https://user-images.githubusercontent.com/1685517/165950648-ee34190b-e25c-4b70-8988-df5a12cf8633.png)